apps/frontend/src/components/cards/controltable/ControlTable.vue

```diff
@@ -142,19 +162,49 @@ export default class ControlTable extends Vue {
   get expand_all() {
     return this.expanded.length === this.items.length;
   }
-
   set expand_all(value: boolean) {
     if(value) {
-      this.single_expand = false;
+      this.singleExpand = false;
       this.expanded = this.items.map((items) => items.key);
     } else {
       this.expanded = [];
     }
   }
 
+  /** Closes all open controls when single-expand is re-enabled */
+  async handleToggleSingleExpand(singleExpand: boolean): Promise<void> {
+    if(singleExpand){
+      this.expandAll = false;
+      this.toggleExpandAllEvaluations()
+    }
+  }
+
+  /** Closes all open controls when single-expand is re-enabled */
+  async handleToggleSyncTabs(syncTabs: boolean): Promise<void> {
+    if(!this.syncTabs){
```

Comment:
Since it's already a parameter here, may as well just use the passed in and not `this.syncTabs`.